### DISCUSSION TITLE
Timeline for Bokeh 2.3.2 support

### DISCUSSION THREAD

**Discussion by votti** [datapane/datapane#116] [Help]
Hi Datapane Team,

Currently plots published via Bokeh 2.3.2 seems not to work for me .
(Warning: Bokeh version 2.3.2 is not supported, your plots may not display correctly, please install version ~=2.2.0)
Published plots wont render.

Unfortunately I have developed my plots before looking into datapane and they wont render on the Bokeh side for Bokeh 2.2

Do you already have a timeline when Bokeh 2.3 should be supported?
Unfortunately I could not find anything about this in the Github issues or on the Website.

Best, Votti

**Comment by mands** [comment]
Hi there,

Yes we were having an issue with Boken 2.3.x - but we'll raise the priority of fixing this and look into it asap.

Thanks!
